app/routes/api.$appId.remove/route.tsx

```diff
@@ -15,7 +15,7 @@ export const action: ActionFunction = async ({ request, params }) => {
   })
 
   if (response.removeEndpoint) {
-    return redirect("/dashboard/apps")
+    return redirect("/account/apps")
```

Comment:
we might have to rethink a lot of these apis. Ideally the api routing should match the account structure so that when we hit it we have both the account and application id context.